sessions/recap-project-6/example-implementation/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "recap-project-6_tourio-app_example-implementation",
+  "name": "recap-project-6_tourio-app",
```

Comment:
I would leave it how it was ` "name": "recap-project-6_tourio-app_example-implementation",`